.github/workflows/publish.yml

```diff
@@ -140,7 +140,7 @@ jobs:
           CHROME_EXT_ID: ${{ secrets.CHROME_EXT_ID }}
           CHROME_REFRESH_TOKEN: ${{ secrets.CHROME_REFRESH_TOKEN }}
       - name: Expose Chrome extension artifact
-        if: '!inputs.prerelease'
+        if: (success() || failure()) && !inputs.prerelease
```

Comment:
May also need to fix (or create the issue) for the VsCode extension publishing failure :)

Reply:
https://github.com/AmadeusITGroup/otter/issues/1931